problem_builder/templates/html/instructor_tool.html

```diff
@@ -9,8 +9,8 @@
     <div class="data-export-field-container">
       <div class="data-export-field">
         <label>
-          <span>{% trans "Username:" %}</span>
-          <input type="text" name="username" />
+          <span>{% trans "Usernames:" %}</span>
+          <input type="text" name="usernames" />
```

Comment:
I think this should explain somehow that if the user is entering multiple usernames, they should be comma-separated. I thought of suggesting placeholder text but I found that it makes it look like the field is filled in, which is confusing. Maybe a tooltip?

![screen shot 2015-08-20 at 4 19 47 pm](https://cloud.githubusercontent.com/assets/945577/9398016/66df110a-4757-11e5-95a8-0f9c97eeee60.png)